libs/common/src/lib/search-widget/search-widget.service.ts

```diff
@@ -194,6 +194,7 @@ export class SearchWidgetService {
 
         let baseSnippet = `<${tagName}${theme}\n  knowledgebox="${kb.id}"`;
         baseSnippet += `\n  ${zone}${features}${prompt}${systemPrompt}${rephrasePrompt}${ragProperties}${ragImagesProperties}${placeholder}${chatPlaceholder}${notEnoughDataMessage}${askToResource}${maxTokens}${maxParagraphs}${queryPrepend}${generativeModel}${vectorset}${filters}${preselectedFilters}${privateDetails}${backend}${jsonSchema}`;
+        baseSnippet += `\n  audit_metadata="{\\"config\\":\\"${currentConfig.id}\\"}"`;
```

Comment:
is this line added on purpose ?

Reply:
Yes, by default we put the search config id in the audit metadata, and the users are free to add more manually after pasting the snippet if they want to.